client/src/services/auth.service.js

```diff
@@ -55,7 +55,11 @@ function updateAccesToken(newAccessToken) {
  */
 async function logout() {
   const { refreshToken } = JSON.parse(localStorage.getItem(LOCAL_STORAGE_KEY));
-  await axios.post("/api/v1/users/logout", { refreshToken });
+  try {
+    await axios.post("/api/v1/users/logout", { refreshToken });
```

Comment:
If you keep this await, the user will not be logged out until the request is complete, I think we could remove it.

Reply:
My aim here is to keep the `await` (to avoid the request being cancelled by the browser), but by ignoring the error if it's happen

Reply:
Ok !